### PR TITLE
Add an example for creating kernel by its name

### PR DESCRIPTION
**Pull request by lisitsyn**
I had to remove functions from ctags due to multiple clash of the 'kernel' name. This way it doesn't clash.

### CODE REVIEW

**Comment by lisitsyn** [comment]
6a0120a85dcdae970b0128776ff992970c-pi

**Comment by karlnapf** [comment]
your machine is wrong ;)

File: tests/unit/base/trained_model_serialization_unittest.cc.py [hunk 1]
```diff
@@ -16,11 +16,13 @@
     'CMultitaskL12LogisticRegression', 'CMultitaskLeastSquaresRegression',
     'CMultitaskTraceLogisticRegression', 'CMultitaskClusteredLogisticRegression',
     'CLatentSVM', 'CLatentSOSVM', 'CDomainAdaptationSVMLinear',
+    'CLinearLatentMachine',
```
Comment (karlnapf) [inline tests/unit/base/trained_model_serialization_unittest.cc.py]: Whats up with these changes?

File: examples/meta/src/base_api/kernel.sg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Kernel k = kernel("GaussianKernel")
```
Comment (karlnapf) [inline examples/meta/src/base_api/kernel.sg]: nice! :)

File: src/shogun/CMakeLists.txt [hunk 3]
```diff
@@ -458,10 +458,11 @@ IF (CTAGS_FOUND)
...
     ADD_CUSTOM_COMMAND(OUTPUT ${CTAGS_FILE}
         COMMAND ${CTAGS_EXECUTABLE} -f ${CTAGS_FILE}
-        # functions, classes, macroses, enumerations, enumerators, typedefs
-        --c++-kinds=fcdgetp
+        # classes, enums, functions
+        --c++-kinds=cgp
         --fields=+im
-        -h "h.hpp"
+        --exclude=*.cpp
+        --languages=c++
```
Comment (lisitsyn) [inline src/shogun/CMakeLists.txt]: @karlnapf this is the solution for ctags

File: tests/unit/base/trained_model_serialization_unittest.cc.py (hunk 1 above)
Comment (lisitsyn) [reply tests/unit/base/trained_model_serialization_unittest.cc.py]: It is kinda wrong but I didn't want to research more on that. For some reason it started generating tests for these ones that are abstract.

Comment (karlnapf) [reply tests/unit/base/trained_model_serialization_unittest.cc.py]: ok well it shouldnt ...

**Pull request merged**